core/src/main/scala/com/salesforce/op/stages/impl/feature/SmartTextVectorizer.scala

```diff
@@ -96,15 +226,19 @@ class SmartTextVectorizer[T <: Text](uid: String = UID[SmartTextVectorizer[T]])(
       isCategorical -> topValues
     }.unzip
 
+    val nameIdentificationResults = if (getRemoveSensitive) Some(detectSensitive(dataset)) else None
+
     val smartTextParams = SmartTextVectorizerModelArgs(
       isCategorical = isCategorical,
       topValues = topValues,
       shouldCleanText = shouldCleanText,
       shouldTrackNulls = $(trackNulls),
-      hashingParams = makeHashingParams()
+      hashingParams = makeHashingParams(),
+      isName = nameIdentificationResults.map(_.isName).getOrElse(Array.empty[Boolean]),
```

Comment:
is name the only type of sensitive information we ever want to hide? perhaps this should be an enum so we leave ourselves room to expand in the future